setup/jest.config.js

```diff
@@ -346,6 +346,7 @@ module.exports = {
     VERSION: '',
     REACT_APP_DEFAULT_NETWORK: 'undefined',
     LISK_ENABLE_DEV_TOOL: false,
+    LISK_DOMAIN: 'https://lisk-test.com',
```

Comment:
dont we need `/send` here as well. Or why is this CONST 3 different things?
` "LISK_DOMAIN": true,` // Boolean
`LISK_DOMAIN: '"https://lisk.com/send"',`  // has `/send`
`LISK_DOMAIN: 'https://lisk-test.com',`

Reply:
Now the lisk.com deployment is updated, @eniolam1000752 will update this and test